### DISCUSSION TITLE
SO web interface stopped working!

### DISCUSSION THREAD

**Discussion by Mav1814** [Security-Onion-Solutions/securityonion#7851] [Unsupported Versions] [answered]
Hello, 
I'm having some trouble to understand why suddenly the web interface of SO stop working. Everything on so-status is ok. Sometimes there are no problems, and other times i cant access web interface. I cant find anything or info about it. I'm lost . 
On nginx error.logs, 
"2022/04/25 12:07:32 [error] 22#22: *207 upstream sent frame for closed stream 1 while reading upstream, client: x.x.x.x, server: x.x.x.x, request: "POST /kolide.launcher.QueryTarget/GetTargets HTTP/2.0", upstream: "grpcs://x.x.x.x.:8080", host: "x.x.x.x"". 
then i cant find anything on the other logs, like soc logs, it seems that everything is working fine. 

The only thing that i found today was, when i tried to make a curl of the web interface of SO, i got this 

"curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above."

Is the first time that i see this. I've been trying to find a way to fix this problem, but i cant find anything. Can anyone help?
cheers!!

**Comment by Mav1814** [comment]
Can anyone help please? I cant find anything or understand what is happening.

**Comment by jertel** [comment, marked as answer]
The curl failure is expected since the TLS certificate is self-signed. 

You stated "the web interface of SO stop working". What _exactly_ are you seeing? Screenshot of the browser will be helpful.

**Reply by Mav1814** [reply]
Hi, the problem is i can't see anything. 
There are times that i can access the web interface and everything works fine and other times like now, i cant, the page doesn't load and then expires. 
On the browser when i try to go to the web interface, it is thinking and then expires. 

About the curl, yeah i guessed it was that.

**Reply by jertel** [reply]
Since you're stating that you "can't see anything" I have to assume that means either the browser window or the entire screen is completely blank (all white, or all black, with no error messages, etc). That scenario would point to a client side problem, possibly with the web browser itself. Try accessing SOC from another device.

If the screen or browser window is not completely blank, then we will need to see exactly what you are seeing to better help you. Screenshots are very helpful in this situation.

**Reply by Mav1814** [reply]
Hi again @jertel , thank you for the response. 
I figure out the problem.. 
It was something I hadn't thought before (my dumbness). Because, since i can access from ssh with vpn, i didnt think of allowing to access the web interface. my bad xD
I forgot to allow my SO(is deployed in my work), for the VPN, since i'm home with COVID-19. 

Again, i'm sorry to make you lose time. And thank you for the quicks responses.
